htdocs/mrp/class/mo.class.php

```diff
@@ -2202,63 +2222,7 @@ public function update(User $user, $notrigger = false)
 	 */
 	public function delete(User $user, $notrigger = false)
 	{
-		global $langs;
-
-		$error = 0;
-		$result = -1;
-
-		// the stockmovements it was created for this line revise
```

Comment:
Is it not the expected behavior to revise the stockmove if the Mo is delete?
if it's not expected then i can take it in a own function?